assets/section-image-banner.css

```diff
@@ -118,10 +168,42 @@
   width: 100%;
 }
 
+@media screen and (min-width: 750px) {
+  .banner--desktop-transparent .banner__box {
+    background-color: transparent;
+    --color-foreground: 255, 255, 255;
+    --color-button: 255, 255, 255;
+    --color-button-text: 0, 0, 0;
+  }
+
+  .banner--desktop-transparent .button--secondary {
+  	--color-button: 255, 255, 255;
+  	--color-button-text: 255, 255, 255;
+  	--alpha-button-background: 0;
+  }
+}
+
+@media screen and (max-width: 749px) {
+  .banner--mobile-bottom::after {
+    display: none;
+  }
+}
+
+.banner::after {
+  content: '';
+  position: absolute;
+  background: #000000;
+  opacity: 0;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+}
+
 .banner__box > * + .banner__buttons {
-  margin: 0 auto;
-  margin-top: 2.3rem;
-  transform: translateX(1rem);
+  align-items: baseline;
```

Comment:
I think we forgot to add `display: flex` because the gap isn't working

```suggestion
  align-items: baseline;
  display: flex;
```

![image](https://user-images.githubusercontent.com/658169/130630164-d4e743a9-bb1f-4431-97ad-7b116f5676ea.png)